custom/icds_reports/templates/icds_reports/icds_app/download.directive.html

```diff
@@ -4,6 +4,7 @@
             <span class="breadcrump-element">Export Program Area Indicators: </span>
         </div>
     </div>
+    <info-message></info-message>
```

Comment:
> Got it. Can you share what the problem is and explain how this addresses it? (or perhaps @Rohit25negi has more context and is comfortable merging)

![a](https://user-images.githubusercontent.com/6906729/64609105-c35cf280-d3e9-11e9-98ba-747d8dc41929.png)

As captured in the image the Tabular report page content was being breaking. I had recently done some changes where I had removed this along with the condition mentioned in the other file.

This is kind of a hack which had been put earlier and me removing it caused the page to break. I tried to find a better solution then this one but only this seems to fix the issue.

Reply:
This is how it should look.

![Screenshot from 2019-09-10 14-05-03](https://user-images.githubusercontent.com/6906729/64609486-ac6ad000-d3ea-11e9-969c-44d5f9abefe0.png)

Reply:
it's because the section title is using `position: fixed` and the subsequent div doesn't have the appropriate spacing set. let me take a quick crack at a more appropriate fix before merging.

Reply:
What a mess. It seems like the `report-content-without-message` and `report-content-with-message` classes are designed to handle this. But for whatever reason they are adding padding to the *bottom* of the div instead of the top so they don't actually properly handle the offsets.

Anyway, I'm nervous about messing with too much base CSS so I guess we can leave this hack in. Can you please add a comment explaining why this is here and linking back to this thread so it can be cleaned up in the future?

Reply:
Offhand does anyone know whether there's any justification for adding padding on the bottom (e.g. we sometimes have a sticky footer) or if those likely never worked properly?

Reply:
actually i'm just going to merge this and then play with it a bit.